webapp/src/services/api/study.ts

```diff
@@ -226,7 +226,7 @@ export const importStudy = async (
   if (onProgress) {
     options.onUploadProgress = (progressEvent): void => {
       const percentCompleted = Math.round(
-        (progressEvent.loaded * 100) / (progressEvent.total || 1),
+        (progressEvent.loaded * 100) / (progressEvent.total ?? 1),
```

Comment:
`progressEvent.total ?? 1` => if `progressEvent.total` equals `0` we will have an infinite value:
`0 ?? 1` => 0
`0 || 1` => 1

So put back `||`.